scio-core/src/main/scala/com/spotify/scio/values/DoubleSCollectionFunctions.scala

```diff
@@ -39,22 +39,26 @@ class DoubleSCollectionFunctions(self: SCollection[Double]) {
   // def sum: SCollection[Double] = this.stats().map(_.sum)
 
   /** Compute the standard deviation of this SCollection's elements. */
-  def stdev: SCollection[Double] = self.transform(_.stats.map(_.stdev))
+  def stdev: SCollection[Double] = self.transform(_.stats.withName("StdDev").map(_.stdev))
```

Comment:
Let's leave transforms in this file alone for now? There're too many of them and it's impossible to catch all. Like there're a few in `histogram()` that we didn't cover?

Plus I doubt these are super useful in streaming. Better to catch them later as they pop up.